content/en/docs/concepts/architecture/cloud-controller.md

```diff
@@ -203,15 +203,15 @@ rules:
 
 ## {{% heading "whatsnext" %}}
 
-[Cloud Controller Manager Administration](/docs/tasks/administer-cluster/running-cloud-controller/#cloud-controller-manager)
+* [Cloud Controller Manager Administration](/docs/tasks/administer-cluster/running-cloud-controller/#cloud-controller-manager)
 has instructions on running and managing the cloud controller manager.
 
-To upgrade a HA control plane to use the cloud controller manager, see [Migrate Replicated Control Plane To Use Cloud Controller Manager](/docs/tasks/administer-cluster/controller-manager-leader-migration/).
+* To upgrade a HA control plane to use the cloud controller manager, see [Migrate Replicated Control Plane To Use Cloud Controller Manager](/docs/tasks/administer-cluster/controller-manager-leader-migration/).
```

Comment:
Maybe we should take this chance to wrap the long lines appropriately.

Reply:
Hi @tengqm , just the `*` or you are recommending to brief the line even more?

Reply:
I mean, wrap the long lines manually, when appropriate. You can break the long lines at the end of a sentence or a punctuation character; you can break them at a maximum line width at 80 or 100 or 120 characters.
Putting all contents on to a single line would make the tracking of future changes very difficult, especially for downstream localization teams.

Reply:
Right , on it!